src/App.vue

```diff
@@ -90,11 +90,28 @@ export default class App extends Mixins(DateMixin) {
   @Action setCurrentDate!: ActionBindingIF
   @Action setCertifyStatementResource!: ActionBindingIF
 
+  // Local Properties
   private filingData: Array<FilingDataIF> = []
+  private draftFiling: IncorporationFilingIF
+  private isDraft: boolean = false
 
-  // Lifecycle event
-  private created (): void {
+  private async created (): Promise<any> {
+    // Mock the nrNumber and Data
+    await this.setNameRequestState({ nrNumber: 'NR7654458', entityType: 'BC', filingId: null })
```

Comment:
Looks like this is missing:
```
@Action setNameRequestState!: ActionBindingIF
```

Also, why is `await` needed here since it doesn't look like this is an async method?

Otherwise, this is fine here for now.

Reply:
OK I'm confused. Is setNameRequestState a store action or a mixin function? It seems to be used inconsistently.

Reply:
You are 100% right, this is confusing. I just threw this in to temporarily simulate a user returning to an incorporation filing in progress, but i left out the Action Binding ( by mistake ). However, the action is still bound to the LegalApiMixin which this class extends which allows ( my mistake ) it to still function.    Either way, good catch. I will add the binding in here for now. Will be removed when the Name Request portion of work is complete. Removing the await as well!